awscli/botocore/eventstream.py

```diff
@@ -585,8 +585,7 @@ def _create_raw_event_generator(self):
         event_stream_buffer = EventStreamBuffer()
         for chunk in self._raw_stream.stream():
             event_stream_buffer.add_data(chunk)
-            for event in event_stream_buffer:
-                yield event
+            yield from event_stream_buffer
```

Comment:
Example of https://docs.astral.sh/ruff/rules/yield-in-for-loop/. Confirmed all these have been changed upstream already.